thinc/model.py

```diff
@@ -277,20 +277,20 @@ def predict(self, X: InT) -> OutT:
         """
         return self._func(self, X, is_train=False)[0]
 
-    def finish_update(self, optimizer: Optimizer) -> None:
+    def finish_update(self, optimizer: Optimizer, **kwargs) -> None:
```

Comment:
I don't think we need to have the `kwargs` argument. The idea is for the parameter-gradients to accumulate somewhere within the `TensorflowShim` object. In the `PyTorchShim`, the gradients accumulate within the `_model` instance, but if that doesn't happen the same way in Tensorflow, we should just store the gradients on the object.

Reply:
Yeah sure will remove that

Reply:
Done. I took the gradients into [self.tensorflow_grads](https://github.com/explosion/thinc/pull/168/files#diff-ea51fecb21b42c0ee05260f800d1583aR53)